PandaGrom/hometask-4/spec/my_hash_spec.rb

```diff
@@ -0,0 +1,68 @@
+# frozen_string_literal: true
+
+require '../my_hash'
+
+# rubocop:disable Metrics/BlockLength
+describe MyHash do
+  let(:hash) { described_class.new }
+
+  describe 'creating a new pair' do
+    before do
+      hash['hello'] = 1
+      hash['world'] = 2
+      hash[4237] = 'code'
+    end
+
+    it 'this should assign values to the keys and show the size of the array according to the number of values' do
```

Comment:
don't use `this` in case descriptions
please, change the naming according to the style guide 
https://github.com/rubocop/rspec-style-guide#naming